docs/src/main/paradox/docs/getting-started/understanding-knowledge-graphs.md

```diff
@@ -1 +1,210 @@
-# Understanding Knowledge Graphs
\ No newline at end of file
+# Understanding Knowledge Graphs
+
+This section will help you understand knowledge graphs and related standards and technologies.
+
+## What is a knowledge graph?
+
+A knowledge graph is a [graph](https://en.wikipedia.org/wiki/Graph_%28abstract_data_type%29) structure that links entities
+together. The links between the entities contain knowledge, based on how the entities connect to each other. For example,
```

Comment:
The links between the entities contain knowledge, based on how they connect to each other.